kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/RealRenderTester.kt

```diff
@@ -213,6 +213,29 @@ internal class RealRenderTester<PropsT, StateT, OutputT : Any, RenderingT>(
     return expected
   }
 
+  private inline fun <reified T : ExpectedWorker<*>> consumeExpectedWorker(
+    predicate: (T) -> Boolean,
+    description: () -> String
+  ): T? {
+    val matchedExpectations = expectations.filterIsInstance<T>()
+        .filter(predicate)
+    return when (matchedExpectations.size) {
+      1 -> {
+        val expected = matchedExpectations[0]
+        // Move the workflow to the consumed list.
+        expectations -= expected
+        consumedExpectations += expected
+
+        expected
```

Comment:
Nit: It looks really weird to just have a variable name sitting by itself. This should have have a `return`.